client/src/components/Forums/ThreadDetails/ThreadDetails.tsx

```diff
@@ -41,17 +43,30 @@ const ThreadDetail: React.FC<ThreadDetailProps> = ({ forumId, threadId }) => {
     setCreatingPost(!creatingPost);
   };
 
+  const handleDeletePost = async (postId: string) => {
+    try {
+      await axios.delete(
+        `/api/forums/${forumId}/threads/${threadId}/posts/${postId}`,
```

Comment:
This could be a ticket to create a function that returns endpoint. It would make the codebase more maintainable _if_ the backend routes changes.

Reply:
Great suggestion @JimmyTran48. I will create some tickets for this.